app/Repository/Advertiser/MySqlStatsRepository.php

```diff
@@ -223,6 +228,46 @@ public function fetchStatsTotal(
         return new Total($calculation, $campaignId);
     }
 
+    public function fetchStatsToReport(
+        string $publisherId,
+        DateTime $dateStart,
+        DateTime $dateEnd,
+        ?string $campaignId = null
+    ): DataCollection {
+        $queryBuilder = (new MySqlStatsQueryBuilder(StatsRepository::STATS_TYPE))
+            ->setAdvertiserId($publisherId)
```

Comment:
Variable name does not match function name. Probable a mistake during adding function - wrong name of parameter $publisherId.